TRIAGING.md

```diff
@@ -0,0 +1,59 @@
+## Issue Triage
+
+When a new issue is opened in a repo in the opensearch-project organization it is automatically labeled as `untriaged`. The maintainers of many active OpenSearch Project repos facilitate [weekly triage meetings](https://www.meetup.com/opensearch/events/) that are open to all, and attendance is encouraged for anyone who hopes to contribute, discuss a new or existing issue, or learn more about the project. Some examples include [core](https://github.com/opensearch-project/OpenSearch/blob/main/TRIAGING.md) and [security triage](https://github.com/opensearch-project/security/blob/main/TRIAGING.md). In addition, a "catch all" weekly triage meeting is held to quickly review untriaged issues in less active repos and ensure that no new issue is left unattended.
+
```

Comment:
How about a section like the following?

> ### Why do triage meetings exist?
> 
> The purpose of the triage meetings is to ensure every issue created in any opensearch-project repository receives a response. Many issues receive immediate engagement and do not need to be specifically triaged. Triage meetings ensure that every issue is filed to the correct place and contains enough information to be actionable. They do not seek to resolve and close all issues, as follow-up discussion and eventual resolution will happen on the issue post itself. Attendance is always welcome, but requesters are not required to attend triage meetings in order for their issue to be addressed.

I'm not 100% sold on this exact text, but I'm wondering if this section can avoid needing to include the "Do I need to attend for my issue to be addressed/triaged?" and "What if I have follow-up questions on an issue?" sections below.

Reply:
Added.

Reply:
I added "What is the purpose of triage meetings?", take a look.

Reply:
Good callout and I like the reframed version of the text - thanks.